ROADMAP.md

```diff
@@ -0,0 +1,22 @@
+# Our Roadmap
+
+Hey, there. Thanks so much for the interest in openSNP and what we're trying to accomplish over the next weeks and months.
+
+We have some ongoing efforts which need to be done as well as some bug fixes and new features we'd love to implement. If you think we should focus on something else: Please just comment, add it or talk to us, whatever you feel like doing!
+
+
+## Ongoing Efforts
```

Comment:
Thanks, I see your point. Do you have a recommendation on how to rephrase it to make it clearer? :-) I'll also try to write a bit about the why for the other points tomorrow. :smile: